packages/wallets/src/sdk.ts

```diff
@@ -25,7 +20,7 @@ type WalletTypeToWallet = {
     "solana-mpc-wallet": SolanaMPCWallet;
 };
 
-class CrossmintWallet {
+export class CrossmintWallet {
```

Comment:
does this not work server side for evm-mpc-wallet?

Reply:
MPC wallets are not yet supported